### PR TITLE
Routes

### PR DESCRIPTION
**Pull request by Sarahshr**

### CODE REVIEW

File: routes/chapters.js [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+  const id = req.params.id;
+  try {
+    timeline = await db.query(
+      `SELECT id, title, first_paragraph, second_paragraph from chapters WHERE id=?`,
+      [id]
+    );
+  } catch (err) {
+    return res.status(500).send("Nope, sorry, I iz broken !");
+  }
+  console.log(timeline[0]);
```
Comment (ThomasMlh) [inline routes/chapters.js]:
```suggestion
```
No no no ! Remove it !

File: routes/chapters.js [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+  return res.json(timeline[0]);
+});
+
+router.post("/chapters", async (req, res) => {
+  try {
+    const response = await db.query(`INSERT INTO chapters SET ?`, [req.body]);
+    res.status(201).send(req.body);
+    console.log(req.body);
```
Comment (ThomasMlh) [inline routes/chapters.js]:
```suggestion
```
Arf !

File: routes/chapters.js [hunk 3]
```diff
@@ -0,0 +1,67 @@
...
+  } catch (err) {
+    res.status(500).send("Nope, sorry, I iz broken !", err);
+  }
+});
+
+router.delete("/chapters/:id", async (req, res) => {
+  let timeline;
+  const id = req.params.id;
+  console.log(id);
```
Comment (ThomasMlh) [inline routes/chapters.js]:
```suggestion
```
?????

File: routes/chapters.js [hunk 4]
```diff
@@ -0,0 +1,67 @@
...
+  try {
+    timeline = await db.query(`DELETE FROM chapters WHERE id=?`, [id]);
+  } catch (err) {
+    console.log(err);
```
Comment (ThomasMlh) [inline routes/chapters.js]:
```suggestion
```
OMG !!!

File: routes/chapters.js [hunk 5]
```diff
@@ -0,0 +1,67 @@
...
+    return res.status(500).send("Nope, sorry, I iz broken !");
+  }
+  return res.status(201).send("deleted successfully");
+});
+
+router.put("/chapters/:id", async (req, res) => {
+  const id = req.params.id;
+  const newTimeline = req.body;
+  console.log(req.body);
```
Comment (ThomasMlh) [inline routes/chapters.js]:
```suggestion
```
Hello there !

File: routes/chapters.js [hunk 6]
```diff
@@ -0,0 +1,67 @@
...
+  try {
+    await db.query(`UPDATE chapters SET ? WHERE id=?`, [newTimeline, id]);
+  } catch (err) {
+    console.log(err);
```
Comment (ThomasMlh) [inline routes/chapters.js]:
```suggestion
```
Wow...

File: server.js [hunk 7]
```diff
@@ -1,21 +1,20 @@
...
-app.use("/auth", require("./routes/auth"));
-
-/* --------------------------------------------------------------------- 404 and server launch */
-app.use((req, res) => {
-  const msg = `Page not found: ${req.url}`;
-  res.status(404).send(msg);
+app.get("/", (req, res) => {
+  console.log("GET / was called!");
+  res.send("Hi there!");
 });
```
Comment (ThomasMlh) [inline server.js]:
```suggestion
```
Maybe it's useless no ? And again a console.log ! 
Maybe you should remove this route

**Review by ThomasMlh — CHANGES_REQUESTED** [review]
Many useless code !
Clean it please !

**Review by ThomasMlh — APPROVED** [review]
It's ok 👍

File: package.json [hunk 8]
```diff
@@ -14,8 +14,10 @@
     "cors": "^2.8.5",
     "dotenv": "^8.2.0",
     "express": "^4.17.1",
-    "jsonwebtoken": "^8.5.1",
+    "express-router": "0.0.1",
     "mysql2": "^2.2.5",
+    "mysql2-promise": "^0.1.4",
```
Comment (LoicBrassart) [inline package.json:19]: Useless, included in `mysql2`

**Review by LoicBrassart — APPROVED** [review]
SnE
(Good reviews bu ThomM, by the way !)

**Pull request merged**